docs/technology/simulation.rst

```diff
@@ -51,6 +51,16 @@ distributions of the SiDBs. Charge distribution surfaces are returned by the SiD
             :members:
 
 
+SiDB Gate design is deemed impossible
+-------------------------------------
+
+**Header:** ``fiction/technology/is_sidb_gate_design_impossible.hpp``
+
+.. doxygenstruct:: fiction::is_gate_design_impossible_params
+   :members:
+.. doxygenfunction:: fiction::is_sidb_gate_design_impossible
+
```

Comment:
This is already documented in `docs/algorithms/design_sidb_gates.rst`. One of them is redundant. Maybe the header was moved?